torch/_logging/_internal.py

```diff
@@ -1084,6 +1085,7 @@ def trace_structured(
     # trace_log never propagates and is ALWAYS DEBUG, so also check that there
     # are handlers instead of checking the log level
     if trace_log.handlers:
+        t0 = time.time()
```

Comment:
time.time uses floating point number, i'd use time.time_ns(), subtract them and then divide by 1000 to log ms number

Reply:
It looks like Scuba prefers second timestamps, c.f. https://fb.workplace.com/groups/scuba.users/permalink/2499634333418500/